static/app/views/projectInstall/issueAlertOptions.tsx

```diff
@@ -178,7 +183,12 @@ class IssueAlertOptions extends DeprecatedAsyncComponent<Props, State> {
     ];
 
     const options: [string, React.ReactNode][] = [
-      [RuleAction.ALERT_ON_EVERY_ISSUE.toString(), t('Alert me on every new issue')],
+      this.props.organization.features.includes('default-high-priority-alerts')
+        ? [
+            RuleAction.ALERT_ON_HIGH_PRIORITY_ISSUES.toString(),
```

Comment:
there's a lot of weird casting to strings happening here, i'll have to take a look at whats going on